src/modules/TitleSearch.js

```diff
@@ -9,20 +8,14 @@ const Search = (props) => {
 
   const [searchResults, setSearchResults] = useState([])
 
-  const handleTitleChange = useCallback(
-    debounce(async (title) => {
+  useEffect(() => {
+    if (!title) return
+
+    setTimeout(async () => {
       const response = await userStory.getStoriesByTitle(title)
       setSearchResults(response.data.data.userStories)
-    }, 600),
-    []
-  )
-
-  useEffect(() => {
-    if (!title) {
-      return
-    }
-    handleTitleChange(title)
-  }, [title, handleTitleChange])
+    }, 600)
+  }, [title])
```

Comment:
why this change?

Reply:
Also @Abhirup-99 , 
as per the discussion [here](https://github.com/facebook/react/issues/19240), they asked to use `useMemo` hook. But for the latest versions, the same warning was generated. Hence the said change was asked to be made.